### PR TITLE
Update "Self-review checklist"

### PR DESCRIPTION
**Pull request by meowius**

### CODE REVIEW

File: content/contributing/collaborating-on-github-docs/self-review-checklist.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-* After opening your pull request, view your changes on staging to confirm the article renders as expected and matches the source. This helps spot issues like typos, content that doesn't follow the style guide, or content that isn't rendering due to versioning problems.
-* Review your changes for technical accuracy.
-* Review your entire pull request to ensure it follows our guidance on creating content that can be translated. For more information, see "[AUTOTITLE](/contributing/writing-for-github-docs/writing-content-to-be-translated)."
-* Check your changes for grammar, spelling, and adherence to the style guide. For more information, see "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide)."
-* If you have added new versioning or made changes to existing versioning, confirm your changes render as expected while viewing each available version of the article.
-* If there are any failing checks in your pull request, troubleshoot them until they're all passing.
+- [ ] If there is a content design plan, your changes meet the user experience and goals outlined in the plan.
```
Comment (meowius) [inline content/contributing/collaborating-on-github-docs/self-review-checklist.md:10]:
```suggestion
* [ ] If there is a content design plan, your changes meet the user experience and goals outlined in the plan.
```

File: content/contributing/collaborating-on-github-docs/self-review-checklist.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+- [ ] Changes have been reviewed for accuracy by a subject matter expert (SME) in the technical area.
```
Comment (meowius) [inline content/contributing/collaborating-on-github-docs/self-review-checklist.md:11]:
```suggestion
* [ ] Changes have been reviewed for accuracy by a subject matter expert (SME) in the technical area.
```

File: content/contributing/collaborating-on-github-docs/self-review-checklist.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+- [ ] The article renders as expected on staging and matches the source. Content is free of errors such as typos and rendering issues due to versioning problems.
```
Comment (meowius) [inline content/contributing/collaborating-on-github-docs/self-review-checklist.md:12]:
```suggestion
* [ ] If there is a content design plan, your changes meet the user experience and goals outlined in the plan.
* [ ] Changes have been reviewed for accuracy by a subject matter expert (SME) in the technical area.
* [ ] The article renders as expected on staging and matches the source. Content is free of errors such as typos and rendering issues due to versioning problems.
```

File: content/contributing/collaborating-on-github-docs/self-review-checklist.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+- [ ] Content follows the guidelines in "[AUTOTITLE](/contributing/writing-for-github-docs/best-practices-for-github-docs)" and "[AUTOTITLE](/contributing/writing-for-github-docs/writing-content-to-be-translated)."
+- [ ] All content has correct grammar and spelling, and adheres to the "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide)."
+- [ ] If you have added new versioning or made changes to existing versioning, your changes render as expected on all versions of the article.
+- [ ] All checks on your pull request are passing. Any failing checks are addressed and are now passing.
```
Comment (meowius) [inline content/contributing/collaborating-on-github-docs/self-review-checklist.md:16]:
```suggestion
* [ ] Content follows the guidelines in "[AUTOTITLE](/contributing/writing-for-github-docs/best-practices-for-github-docs)" and "[AUTOTITLE](/contributing/writing-for-github-docs/writing-content-to-be-translated)."
* [ ] All content has correct grammar and spelling, and adheres to the "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide)."
* [ ] If you have added new versioning or made changes to existing versioning, your changes render as expected on all versions of the article.
* [ ] All checks on your pull request are passing. Any failing checks are addressed and are now passing.
```

**Comment by meowius** [comment]
Moving to `docs-internal` 👍

**Comment by Ajiiip** [comment]
[Uploading google_privacy_policy_ms (1).pdf…]()
[Uploading Managing Extensions in Your Enterprise_salinan.pdf…]()

**Pull request closed without merging**